documentation/asciidoc/computers/raspberry-pi/eeprom-bootloader.adoc

```diff
@@ -481,6 +518,14 @@ PARTITION=2
 
 Default: 0
 
+[[PARTITION_WALK]]
+==== `PARTITION_WALK`
+This property is designed to improve the reliability of `A/B` boot schemes using `autoboot.txt` by searching for boot-able partitions if the specified partition does not appear to be bootable. If `PARTITION_WALK=1` and the selected partition is not bootable and does not have a valid `autoboot.txt` then the bootloader will check each partition in turn (up to 8 and wrapping to 0) to see if it is bootable (contains `start.elf` or `config.txt` and a suitable device-tree on Pi5 or newer).
```

Comment:
It's just a gut-feeling, but IMHO "the selected partition" sounds like weird phrasing here. Perhaps it works better using "the requested partition" phrasing from the sentence below? :shrug:

Reply:
ok makes sense